src/manifolds/ProbabilitySimplex.jl

```diff
@@ -32,7 +32,7 @@ This implementation follows the notation in [^ÅströmPetraSchmitzerSchnörr2017
 
 # Constructor
 
-    ProbabilitySimplex(n::Int; closed::Bool=false)
+    ProbabilitySimplex(n::Int; closed::Symbol=:open)
```

Comment:
I would prefer to call the variable then `boundary` and not `closed`?

Reply:
Sure, makes sense.